docs/asciidoc/advancedFeatures.adoc

```diff
@@ -193,21 +193,21 @@ Compare the first example to the second example, shown in the following code:
 ====
 [source,java]
 ----
-Sink<String> hotSource = Sinks.many().multicast().onBackpressureError();
+Sinks.Many<String> hotSource = Sinks.many().unsafe().multicast().directBestEffort();
 
 Flux<String> hotFlux = hotSource.asFlux().map(String::toUpperCase);
 
-
 hotFlux.subscribe(d -> System.out.println("Subscriber 1 to Hot Source: "+d));
 
-hotSource.next("blue")
-         .next("green");
+hotSource.emitNext("blue", FAIL_FAST);
+//note: orThrow is an alternative to emitNext+Sinks.EmitFailureHandler.FAIL_FAST above, suitable for tests
+hotSource.tryEmitNext("green").orThrow();
```

Comment:
I would have inverted those two lines as `orThrow` is used after the comment and though one may not really understand where does it comes from

Reply:
I updated to use a callout. found a bad callout in an unrelated asciidoc file and fixed it as a separate commit, so I rebased the PR into 2 commits FYI